gen3/bin/kube-roll-all.sh

```diff
@@ -18,7 +18,7 @@ gen3_load "gen3/lib/kube-setup-init"
 
 # Set flag, so we can avoid doing things over and over
 export GEN3_ROLL_ALL=true
-
+gen3 db init
```

Comment:
When was this run before? Was the user running it before the roll all?

Reply:
Im not sure exactly where is gets run but it eventually is run in one of the scripts that gen3 roll all calls

Reply:
More specifically gen3 db init is run before the first call to the gen3 db script and if JENKINS_HOME is set but there can be a mix up with the function because if /g3auto/dbfarm/servers.json is created before running gen3 db init then the function wont create the secret dbfarm-g3auto.